.gitignore

```diff
@@ -4,3 +4,5 @@
 __pycache__/
 config.ini
 data
+.idea/
+serenata-de-amor.iml
```

Comment:
Sorry @cuducos ! At beginning I was using IntelliJ IDE and this file is generated automatically with some IDE project's properties.

Do you want me to remove that?